### DISCUSSION TITLE
v0.2.0

### DISCUSSION THREAD

**Discussion by argenos** [argenos/zotero-mdnotes#183] [General]
## What's Changed
* Use joplin-turndown for HTML->MD conversion by @nathanlesage in https://github.com/argenos/zotero-mdnotes/pull/38
* Added Date Fix by @PhilipMottershead in https://github.com/argenos/zotero-mdnotes/pull/67
* Update formatting.md by @einalex in https://github.com/argenos/zotero-mdnotes/pull/89
* Clearer text by @Jmuccigr in https://github.com/argenos/zotero-mdnotes/pull/93
* (docs) fix error in formatting.md by @atraphaxis in https://github.com/argenos/zotero-mdnotes/pull/159
* docs: replace {{noteTitle}} with {{title}} by @Glunk in https://github.com/argenos/zotero-mdnotes/pull/145
* fix bug on filename when running on Windows by @rxy0424 in https://github.com/argenos/zotero-mdnotes/pull/166
* Bug fix messing url parsing for obsidian links on Windows by @knavynde in https://github.com/argenos/zotero-mdnotes/pull/108
* add locale support for zh-CN by @xfap in https://github.com/argenos/zotero-mdnotes/pull/163

## New Contributors
* @nathanlesage made their first contribution in https://github.com/argenos/zotero-mdnotes/pull/38
* @PhilipMottershead made their first contribution in https://github.com/argenos/zotero-mdnotes/pull/67
* @einalex made their first contribution in https://github.com/argenos/zotero-mdnotes/pull/89
* @Jmuccigr made their first contribution in https://github.com/argenos/zotero-mdnotes/pull/93
* @atraphaxis made their first contribution in https://github.com/argenos/zotero-mdnotes/pull/159
* @Glunk made their first contribution in https://github.com/argenos/zotero-mdnotes/pull/145
* @rxy0424 made their first contribution in https://github.com/argenos/zotero-mdnotes/pull/166
* @knavynde made their first contribution in https://github.com/argenos/zotero-mdnotes/pull/108
* @xfap made their first contribution in https://github.com/argenos/zotero-mdnotes/pull/163

**Full Changelog**: https://github.com/argenos/zotero-mdnotes/compare/0.1.3...0.2.0

<hr /><em>This discussion was created from the release <a href='https://github.com/argenos/zotero-mdnotes/releases/tag/0.2.0'>v0.2.0</a>.</em>

**Comment by Explorare** [comment]
Failed to install on Zotero v6.0.6 with this error message.
>The add-on "Mdnotes for Zotero" could not be installed. It may be incompatible with this version of Zotero.

**Reply by argenos** [reply]
Should work now with the 0.2.3 release. Sorry about that!

**Comment by cjpoor** [comment]
Thanks for the update :-)
